docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
+2. User accesses logs
+3. User looks for a stacktrace with their App’s processes name
+4. User finds stacktrace + root cause and begins to fix
+5. User commits fix
+6. User’s Build passes
+7. User merges PR
+#### Use Case
+1. GitHub has Status update and comment from fastlane on PR indicating that the build failed
+2. User opens fastlane ci Web UI
+4. Assuming already logged in
```

Comment:
you can use `1.` for all bullets and it will auto-increment 🎉

Reply:
Yes, always using `1.` will make our lives so much easier, as it allows us to easily change the order also 👍 Markdown is magic

Reply:
donezo